terraform/files/bin/deploy_harbor.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+. ~/bin/cccfg.inc
+export KUBECONFIG=${KUBECONFIG_WORKLOADCLUSTER}
```

Comment:
`cccfg.inc` contains `KCONTEXT` so instead of exporting KUBECONFIG just `kubectl $KCONTEXT` could be used (probably a "safer" option)

Reply:
I added it back in 6161a157bb1c78d19a521259c1c0477fccab7c04, because scripts for generating harbor secrets need it